src/Parser.js

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Better naming: DeploymentConfigParser for example
Put it in a parser sub folder